sdk/core/Cargo.toml

```diff
@@ -13,24 +13,26 @@ categories = ["api-bindings"]
 edition = "2018"
 
 [dependencies]
+async-trait = "0.1"
+bytes = "1.0"
 chrono = "0.4"
-http = "0.2"
+clap = "2.33.3"
+dyn-clone = "1.0"
 futures = "0.3"
+http = "0.2"
 hyper = { version = "0.14", optional = true }
+hyper-rustls = { version = "0.22", optional = true }
 log = "0.4"
-thiserror = "1.0"
+oauth2 = "4.0.0"
+rand = "0.7"
+reqwest = { version = "0.11", features = ["stream"], optional = true }
+rustc_version = "0.3.3"
```

Comment:
What code is requiring `rustc_version`?

Reply:
IIRC, this is also just reading an env var and I can redefine it. As I mentioned to @rylev I figured it was all statically linked anyway, but if these are larger build-time pulls I can replace them. I know clap is larger. I can check this one.

That said, the cargo variables probably won't work anyway because they'll be for the users' crates being built. I'm going to play around with a few ideas and see what I can make work here.

Reply:
Good thing I looked into this more. Turns out it's implemented by running `$RUSTC --version` and parsing the output, and is really only meant for use in build scripts anyway. So that's what I did after failing to find any other possibilities. I'm surprised rustc doesn't set some env var itself for versioning branches like MSVC does.